### PR TITLE
api maps and facebook

### PR DESCRIPTION
**Pull request by amityahalom**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const users = require("./routes/users")  // create route for every model
 const login = require("./routes/login")
 const orders = require("./routes/orders");
 const concerts = require("./routes/concerts");  // create route for every model
+const placesRoutes = require("./routes/places");
```
Comment (MayaHarel1) [inline app.js:6]: change to places

File: app.js [hunk 2]
```diff
@@ -24,5 +25,6 @@ server.use('/api_users', users) // for (x,y) - when you get x, do y
 server.use('/api_login', login)
 server.use('/api_orders', orders)
 server.use('/api_concerts', concerts) 
+server.use('/api_places', placesRoutes);
```
Comment (MayaHarel1) [inline app.js:28]: here too

File: controllers/concerts.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const concertsService = require("../services/concerts")
 const { getArtistLatestAlbum } = require('../services/spotifyService');
 const multer = require("multer");
+const facebookService = require('../services/facebookService.js');
```
Comment (MayaHarel1) [inline controllers/concerts.js:4]: why js?

File: controllers/places.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+const placesService = require("../services/places.js")
```
Comment (MayaHarel1) [inline controllers/places.js:1]: without js

File: public/scripts/showConcert.js [hunk 5]
```diff
@@ -1,61 +1,52 @@
 // Function to fetch concert details
-async function getConcert() {
+function getConcert() {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
-    const id = urlParams.get("id");
```
Comment (lior610) [inline public/scripts/showConcert.js:5]: החזרנו ל-ID

File: services/facebookService.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+const axios = require('axios');
+
+// Facebook Page Access Token
```
Comment (lior610) [inline services/facebookService.js:3]: env.local

**Pull request merged**